README.md

```diff
@@ -70,6 +70,30 @@ built client from the client repository. Please consult the client's
 documentation for instructions on building the client in a development
 environment.
 
+**Tip**: If you get a **permission denied** error when running `npm link`
```

Comment:
I appreciate that this documentation was just moved from h.readthedocs.io but in general I'm wary of including generic documentation _for 3rd-party tools_. In this context for example, a search for 'npm link permission denied' turns up [this more detailed and up to date official documentation](https://docs.npmjs.com/getting-started/fixing-npm-permissions) on the subject, which we could just link to instead.

Thoughts?